src/main/java/com/jlr/ttl/poc/loadmanager/models/Vehicle.java

```diff
@@ -11,9 +18,14 @@ public class Vehicle {
     @Column(name = "loc_code")
     private String loc;
 
-    public Vehicle(Integer vin, String loc_code) {
+    @Column(name = "status")
+    private String status;
+
+    public Vehicle(Integer vin, String loc_code, Status status) {
```

Comment:
Remove constructors and getters and setters. Use the following annotations instead:

AllArgsConstructor
NoArgsConstrutctor
Data
Build